### ISSUE TITLE
Query regarding mIoU reproduction results

### ISSUE THREAD

**Issue by yanggao2000** [boschresearch/GaussianFlowOcc#4]
Dear authors,

Thank you for your excellent work on GaussianFlowOcc and for making the source code publicly available. It's a very insightful paper!

We've been reproducing your work, following the paper's setup (official hyperparameters, 4xA100s). Our final result is an mIoU of 15.19 (see screenshot for more details), compared to the 17.08 reported in the paper.

<img width="757" height="553" alt="Image" src="https://github.com/user-attachments/assets/78cbc631-eabc-4034-97a8-9722e604cc4e" />

We were wondering if this ~1.9 point difference is within an expected range of variance? 
Any insight you could offer would be very helpful. Thanks a lot!

**Comment by SBoeder** [comment]
Hi @yanggao2000, 
can you try and run the eval with the command line argument `--nbh 4` and/or  `--nbh 5` and report the results? This parameter sets the maximum number of neighboring voxels to be considered by each Gaussian (default ins this repo is 3 currently, but I used 4 in my experiments). This should increase the accuracy while reducing the FPS. I will update the code to set this to 4 by default and adjust the Readme.

**Comment by yanggao2000** [comment]
Dear @SBoeder ,
Thank you for the quick reply!
I just evaluated it with --nbh 4/5, and it gave 'mIoU_0.05': 15.5/15.61. They are better than before, but still worse than the expected range. Do I need to retrain the model with a similar argument change?
Thank you again!

**Comment by SBoeder** [comment]
Dear @yanggao2000,

thanks for reporting the numbers. There is no need to retrain a model, as this only affects voxelization, which is done only for evaluation. I will have to try to reproduce the results myself and see what could be the reason. I can imagine that there is additional noise introduced by pseudo-label generation, which is also not completely deterministic. I will report back once I have new results. Thanks!

**Comment by jxl152** [comment]
> Hi, the best results reported in the paper have been achieved by using a different loss weighting than what is currently set in the config of this repository. Also, by adding the "--nbh 4" or "--nbh 5" flag for the evaluation script (no need to re-train), the influence range of each Gaussian during voxelization is increased (default is --nbh 3). This should lead to better results. I will update this repo ASAP.

I started training using the current config. I look forward to the updated version! Thanks!

**Comment by SBoeder** [comment]
Hello everyone,

I just pushed these changes to be default, so there is no need to change anything in the config manually anymore. Just running training with the updated loss weighting and running the eval should produce better results. You can further increase the influence of the Gaussians if you want to test it out. Simply add `--nbh 5` or more, the default is 4 now. This is also described in the README now. Thanks everyone!

**Comment by seamie6** [comment]
> Hello everyone,
> 
> I just pushed these changes to be default, so there is no need to change anything in the config manually anymore. Just running training with the updated loss weighting and running the eval should produce better results. You can further increase the influence of the Gaussians if you want to test it out. Simply add `--nbh 5` or more, the default is 4 now. This is also described in the README now. Thanks everyone!

I see a lot of files have been updated in the repo. Is adding ```depth_lw=.05, sem_lw=2``` to the Gaussian Rasterizer and the increased neighbor count during voxelization all that's needed for results seen in the paper? Just don't want to rewrite files that I have modified. Thanks
